apps/api/docs/api-test-cases.md

```diff
@@ -52,13 +52,13 @@ Before running the test cases, make sure you have the following prerequisites in
     1. For console output
 
        ```bash
-       newman run osmo-notify.postman_collection.json -e osmo-notify.postman_environment.json
+       newman run osmoX.postman_collection.json -e osmoX.postman_environment.json
```

Comment:
You can fix the casing for this as `OsmoX.postman_environment.json`

Reply:
Addressed in https://github.com/OsmosysSoftware/osmo-x/pull/152/commits/ffca8627cb4ae9663e16c6d1288440193d1ada1b